.github/workflows/kind-cluster-custom-resources.yaml

```diff
@@ -0,0 +1,148 @@
+# Copyright 2022 The Sigstore Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Test policy-controller with custom resource
+
+on:
+  pull_request:
+    branches: [ 'main', 'release-*' ]
+
+defaults:
+  run:
+    shell: bash
+
+permissions: read-all
+
+jobs:
+  cip-test-trustroot-bring-your-own-keys:
+    name: ClusterImagePolicy e2e tests with custom resources
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false # Keep running if one leg fails.
+      matrix:
+        k8s-version:
+          - v1.27.x
```

Comment:
@cpanato is updating the tested k8s versions, should proabably bump these once it merges.

Reply:
@cpanato - Could you please let me know when you merge your changes so i can make the right adjcments here?

Reply:
This was done, so go ahead and adjust this matrix.

Reply:
Updated to:
- v1.29.x
- v1.30.x
- v1.31.x
- v1.32.x